### PR TITLE
INTEGRA-3927: add SHA digest for jar versions

### PR DESCRIPTION
**Pull request by asargsyan999**
sha 256 digest versioning added to track identify previous changes.

NTEGRA-3927

### CODE REVIEW

File: java/sha_digest.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+VERSION="3.0.0"
```
Comment (skharab-anaplan) [inline java/sha_digest.sh]: I think we need to read this from POM.

File: java/sha_digest.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
+FILE_NAME=sha-digest-version.txt
+CURRENT_DATE_TIME=$(date '+%Y-%m-%dT%T.%zZ')
+SHA_256_VALUE=$(cd anaplan-connect-client/target; openssl dgst -sha256 anaplan-connect-client-${VERSION}-jar-with-dependencies.jar | cut -d=  -f2)
```
Comment (skharab-anaplan) [inline java/sha_digest.sh]: We need to add this in release steps that once the branch is merged to master, run a mvn clean install / package and create a sha of that jar.

**Review by skharab-anaplan — COMMENTED** [review]
Comments

File: java/sha_digest.sh (hunk 1 above)
Comment (asargsyan999) [reply java/sha_digest.sh]: done

File: java/sha_digest.sh (hunk 2 above)
Comment (asargsyan999) [reply java/sha_digest.sh]: done

**Review by skharab-anaplan — APPROVED** [review]
APPROVED

**Pull request merged**